source/sass/component/_slider_item.scss

```diff
@@ -28,11 +28,13 @@ $c-slider-item-container-max-width: calc(#{$base} * 118) !default;
         margin: 0;
     }
 
-    &:focus-visible {
+    &:focus-visible,
+    .c-card:focus {
```

Comment:
I think this would be more suitable in c-card file. To enshure that we dosen't 'cross reference' other components. 

This may be achived by adding a modifier to card. 

c-card--focus-inset